src/init/pkg_installer.sh

```diff
@@ -48,22 +33,14 @@ if [ "$status" = "connected" -a $RESULT -eq 0  ]; then
     echo -ne "0" > ${WAZUH_HOME}/var/upgrade/upgrade_result
     echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade finished successfully." >> ${WAZUH_HOME}/logs/upgrade.log
 else
-    # Restoring backup
-    echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade failed. Restoring..." >> ${WAZUH_HOME}/logs/upgrade.log
+    echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade failed..." >> ${WAZUH_HOME}/logs/upgrade.log
 
     CONTROL="$WAZUH_HOME/bin/wazuh-control"
     if [ ! -f $CONTROL ]; then
         CONTROL="$WAZUH_HOME/bin/ossec-control"
     fi
 
-    $CONTROL stop >> ${WAZUH_HOME}/logs/upgrade.log 2>&1
-    tar xzf ${WAZUH_HOME}/backup/backup_${WAZUH_VERSION}_[${BDATE}].tar.gz -C / >> ${WAZUH_HOME}/logs/upgrade.log 2>&1
+    $CONTROL start >> ${WAZUH_HOME}/logs/upgrade.log 2>&1
```

Comment:
I think that you can add a log to indicate that Wazuh will be started in the state it is at this point.

Reply:
done